src/common/tracer.h

```diff
@@ -62,10 +96,20 @@ class Value {
 
 using jspan_attribute = Value;
 
-struct jspan_context {
-  jspan_context() {}
-  jspan_context(bool sampled_flag, bool is_remote) {}
+namespace opentelemetry {
+inline namespace v1 {
+namespace trace {
+class SpanContext {
```

Comment:
would like to avoid a) declaring types inside of the `opentelemetry` namespace, and b) exposing types from the `opentelemetry` namespace in public librados headers

but we can revisit this in a followup PR if you prefer to merge as-is